taskapp/views.py

```diff
@@ -70,7 +71,8 @@ def put(self, request, pk, *args, **kwargs):
 
 class FeedbackView(APIView):
     permission_classes = [permissions.IsAuthenticated]
-
+    
+    @admin_required

```

Comment:
when you use decorator then doesn't need to check role in the API also.
remove "if request.user.role != Customer.MANAGER"